src/main/kotlin/no/nav/familie/ef/sak/vedtak/uttrekk/UttrekkController.kt

```diff
@@ -0,0 +1,34 @@
+package no.nav.familie.ef.sak.vedtak.uttrekk
+
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import java.util.UUID
+
+/**
+ * Skal vi returnere en liste med FNR ? link til behandling?
+ * Hvordan gjør vi med de som er kode 6/7?
+ */
+
+@RestController
+@RequestMapping(path = ["/api/uttrekk"])
+@ProtectedWithClaims(issuer = "azuread")
+@Validated
+class UttrekkController(private val uttrekkVedtakService: UttrekkArbeidssøkerService) {
```

Comment:
Notere att det ikke er noen tilgangskontroll her.
Det skjer en tilgangskontroll inne i servicen vid OPPDATERING av kontrollert. 

API'et returnerer kun metadata, uten fnr eller kobling til en person. 
Hvis vi senere velger å gå for å kunne legge inn en beskrivelse må vi legge til tilgangskontroll på uthenting